src/remove-commonjs-allowed-dependencies.ts

```diff
@@ -0,0 +1,59 @@
+import {cloneDeep} from 'lodash-es';
+
+import {AngularVersion} from './types/angular-version';
+import {AngularJson} from './types/angular-json';
+import {angularVersionComparer} from './angular-version-comparer';
+import {ProjectConfiguration} from './types/project-configuration';
+import {ArchitectConfiguration} from './types/architect-configuration';
+
+const ExclusiveAngularVersion = '10.0.x';
```

Comment:
suggestion: use lower camel case
```suggestion
const exclusiveAngularVersion = '10.0.x';
```